strings/trainer_sliding_strings.string

```diff
@@ -1,41 +1,202 @@
+
+#org @sText_Kurtis_LastSwitchIn
+Team Pluto won't fall to the likes\nof you. Not now, not ever.\p 
+
+#org @sText_Kurtis_LowHP
+Now you're making me angry!\p 
+
+// Alistair
+#org @sText_Alistair_FirstMonDown
+Impressive. But you can't win.\p 
+
+#org @sText_Alistair_LastSwitchIn
+Don't try to stop me! I'm doing\nthis for everyone's benefit!\p 
+
+#org @sText_Alistair_LowHP
+Ngh. My perfect world... I can't\nlose like this!\p 
+
+// E4 Hannah
+#org @sText_Hannah_FirstMonDown
+Impressive, trainer! But you'll\nfind the Elite Four are a cut\nabove regular Gym Leaders!\p 
+
+#org @sText_Hannah_LastSwitchIn
+You really are something, I'll\ngive you that!\p 
+
+#org @sText_Hannah_LowHP
+We're not done yet!\p 
+
+// E4 Liam
+#org @sText_Liam_FirstMonDown
+Woo, look at you go! I'm just\ngetting started, though.\p 
+
+#org @sText_Liam_LastSwitchIn
+Your challenge ends here! It's\nnothing personal, just my job.\p 
+
+#org @sText_Liam_LowHP
+Unbelievable! It's been ages since\na challenger has done this well.\p 
+
+// E4 Jasmine
+#org @sText_Jasmine_FirstMonDown
+We're just getting warmed up. Are\nyou ready?\p 
+
+#org @sText_Jasmine_LastSwitchIn
+The battle's really heating up\nnow, wouldn't you agree?\p 
 
-#org @sText_MirskleFirstMonDown
-Whoaaaaa[.]\nThat was intense[.]\p 
+#org @sText_Jasmine_LowHP
+Oh no! We've really slipped up,\nhaven't we?\p 
 
-#org @sText_VegaLastSwitchIn
-Prepare yourself for my trump card.\p 
+// E4 Thomas
+#org @sText_Thomas_FirstMonDown
+You've defeated my first Pokémon?\nI won't let it happen again!\p 
 
-#org @sText_AliceLastSwitchIn
-It's time for you to fly high,\n[FD][06]!\p 
+#org @sText_Thomas_LastSwitchIn
+It's over now! I won't allow you\nto waste Miss Selene's time.\p 
 
-#org @sText_MelLastSwitchIn
-Fight like the roaring waterfall\nbehind us, [FD][06]!\p 
+#org @sText_Thomas_LowHP
+Miss Selene, forgive me.\p 
 
-#org @sText_GalavanLastSwitchIn
-Ha ha ha ha!\nIt ain't over yet!\p 
+// Champion
+#org @sText_Champion_FirstMonDown
+Impressive. I feel we're going to have\na really exciting battle today!\p 
 
-#org @sText_TessyLastSwitchIn
-It's tahm for the real fight,\nI reckon'!\p
+#org @sText_Champion_LastSwitchIn
+It's been a long time since I've had a\nbattle this close!\p 
 
-#org @sText_Zeph1LowHP
-Impossible[.]\p 
+#org @sText_Champion_LowHP
+We won't give up until the very end!\nYou'd better give it your all too!\p 
 
-#org @sText_Zeph2LowHP
-No!\nI can't lose again!\p 
+// Champion Rematch
+#org @sText_ChampionRematch_FirstMonDown
+This really takes me back to our\nfirst battle. Do you remember it,\ntoo?\p 
 
-#org @sText_MaximaLastSwitchIn
-The power of Mega Evolution[.]\nCan you feel it within your reach?\p 
+#org @sText_ChampionRematch_LastSwitchIn
+It's quite refreshing to see how\nstrong a bond you share with your\nPokémon.\p 
 
-#endif
+#org @sText_ChampionRematch_LowHP
+I may not be the champion any\nlonger, but it wouldn't be very\nsporting for me to give up now!\p 
```

Comment:
" but it wouldn't be very sporting for me to give up now!" sounds off to me

Reply:
Changed it up a bit.  LMK what you think